flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/windows.scala

```diff
@@ -150,7 +147,7 @@ class TumblingWindow(size: Expression) extends GroupWindow {
   def as(alias: String): TumblingWindow = as(ExpressionParser.parseExpression(alias))
 
   override private[flink] def toLogicalWindow: LogicalWindow =
-    ProcessingTimeTumblingGroupWindow(alias, size)
+    ProcessingTimeTumblingGroupWindow(name, size)
```

Comment:
Hi, @shaoxuan-wang  thanks a lot for the review. I have updated the PR according to your comments. The change list:
 1. Remove GroupWindowedTable.
 2. Chanage "name" to "alias".
 3. Add testMultiWindow.
 4. Rebase code and fixed the conflicts.